website/package.json

```diff
@@ -37,7 +37,7 @@
     "@marsidev/react-turnstile": "^0.0.7",
     "@next-auth/prisma-adapter": "^1.0.5",
     "@next/bundle-analyzer": "^13.2.4",
-    "@next/font": "^13.2.4",
+    "@next/font": "13.0.6",
```

Comment:
Is there any reason that you have pin the version?

Reply:
To match the version of `next` that is currently also using the same version (also pinned to 13.0.6)

The same was also done for `eslint-config-next` and `next` itself. The previously installed version was incompatible  and would result in error in dev and/or builds.

Reply:
So we need to update next then, it's been long time since our last deps bump